OUXT-Polaris/robotx_packages
# [Add] Travis CI Integration

## Status
Add the Travis CI Integration

## Description
The package be able to do build test.
You can check that result on [Travis page](https://travis-ci.org/OUXT-Polaris/robotx_packages).
And also, the integration between github and Travis CI is automatic.

## Related PRs
No related PRs

## Todos
- [x] Tests (Check if the all of Travis CI test are passed.)

## Steps to Test or Reproduce
1. Just Push or your commits
2. Check the result of build tests on [Travis page](https://travis-ci.org/OUXT-Polaris/robotx_packages)

You can access to Travis page by clicking the icon on the github page,
and also clicking the badge on README.
